src/faces.jl

```diff
@@ -499,19 +499,20 @@ function Base.merge(a::Face, b::Face)
         # to narrow the types in e.g. `aheight * bheight`
         aheight = a.height
         bheight = b.height
-        Face(ifelse(isnothing(b.font),          a.font,          b.font),
-             if isnothing(bheight)      aheight
-             elseif isnothing(aheight)  bheight
-             elseif bheight isa Int     bheight
-             elseif aheight isa Int     round(Int, aheight * bheight)
-             else aheight * bheight end,
-             ifelse(isnothing(b.weight),        a.weight,        b.weight),
-             ifelse(isnothing(b.slant),         a.slant,         b.slant),
-             ifelse(isnothing(b.foreground),    a.foreground,    b.foreground),
-             ifelse(isnothing(b.background),    a.background,    b.background),
-             ifelse(isnothing(b.underline),     a.underline,     b.underline),
-             ifelse(isnothing(b.strikethrough), a.strikethrough, b.strikethrough),
-             ifelse(isnothing(b.inverse),       a.inverse,       b.inverse),
+        abheight = if isnothing(bheight) aheight
```

Comment:
I have seen that pattern a bit, but I can't say I'm a fan of it. Perhaps it's a bit selfish of me, but I'm rather a fan of `(if cond val otherwise)` and `(cond ...)` forms and so like to use them over ternaries in projects where I'm the primary author.

I'm also a fan of consistency, and there are nine other one-line `if ... else ... end` statements floating around already.